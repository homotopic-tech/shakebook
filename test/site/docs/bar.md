---
title: The Bar in the mountains
description: Hi im champ
date: 2020-06-01
---

Lorem ipsum
