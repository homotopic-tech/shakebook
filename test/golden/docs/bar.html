<html>
  <head>
    <title>The Bar in the mountains</title>
  </head>
  <body>
    <h1>This post page is called The Bar in the mountains</h1>
    <p>It was generated from the source test/site/docs/bar.md</p>
    <p>For the url </p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the recent posts again:
      <ul>
      </ul>
    </p>
  </body>
</html>
